src/GafferScene/LightToCamera.cpp

```diff
@@ -134,6 +134,12 @@ const char *lightType( const IECore::CompoundData *shaderParameters, const std::
 		return "";
 	}
 
+	if( boost::starts_with( metadataTarget, "light:" ) && shaderParameters->member<FloatData>( "shaping:cone:angle" ) )
```

Comment:
This is definitely a step forward for USD lights authored within Gaffer, where we only expect `shaping:cone:angle` to have been enabled if it is wanted and has a sensible value. But I've seen at least one other DCC that seemed to always author `shaping:cone:angle`, using a value of 90 (or was it 180?) as a sentinel value meaning "ignore me". I don't know what the official line on that is as UsdLux is badly under specified, but I think it'd be worth us ignoring values `>=90` here regardless, since the perspective projection is broken in those cases. I'm going to merge anyway as we wanted to include this in today's release, but I think it might be worth doing a quick follow up?